src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -123,6 +125,18 @@ public HttpRequestHeaders Headers
             }
         }
 
+        public HttpRequestOptions Options
+        {
+            get
+            {
+                if (_options == null)
+                {
+                    _options = new HttpRequestOptions();
+                }
+                return _options;
+            }
+        }
```

Comment:
Nit: the implementation here can be simplified to be:
```C#
public HttpRequestOptions Options => _options ??= new HttpRequestOptions();
```

Reply:
Addressed in recent commits